src/pages/RepoPage/ComponentsTab/Header/BranchSelector/BranchSelector.tsx

```diff
@@ -21,7 +22,11 @@ const defaultQueryParams = {
 const getDecodedBranch = (branch?: string) =>
   !!branch ? decodeURIComponent(branch) : undefined
 
-const BranchSelector: React.FC = () => {
+interface BranchSelectorProps {
+  isDisabled: boolean | undefined
+}
+
+const BranchSelector: React.FC<BranchSelectorProps> = ({ isDisabled }) => {
```

Comment:
```suggestion
const BranchSelector: React.FC<BranchSelectorProps> = ({ isDisabled = false }) => {
```

Adding a default value might make this a bit easier to use. Nit.